sno/sqlalchemy/create_engine.py

```diff
@@ -16,6 +16,8 @@
 from sno.geometry import Geometry
 from sno.exceptions import NotFound, NO_DRIVER
 
+GPKG_CACHE_SIZE_MB = 200
```

Comment:
make it in MiB, then the maths below makes sense (the pragma value is in KiB)

Reply:
Done